src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -111,13 +122,13 @@ export module DotNet {
 
   /**
    * Creates a JavaScript object reference that can be passed to .NET via interop calls.
-   * 
+   *
    * @param jsObject The JavaScript Object used to create the JavaScript object reference.
    * @returns The JavaScript object reference (this will be the same instance as the given object).
    * @throws Error if the given value is not an Object.
    */
   export function createJSObjectReference(jsObject: any): any {
-    if (jsObject instanceof Object) {
+    if (jsObject && typeof jsObject === 'object') {
```

Comment:
This change is needed because an ES6 module isn't `instanceof Object`, but we do want such things to be usable as `JSObjectReference` values.